diff --git a/lib/handler/httphandler.h b/lib/handler/httphandler.h
index db3fd56..815a310 100644
--- a/lib/handler/httphandler.h
+++ b/lib/handler/httphandler.h
@@ -14,9 +14,9 @@
 #include "settingshandler.h"
 #include "websockethandler.h"
 #include "xtpwebhandler.h"
-#include "httpServer/httpServer.h"
-#include "httpServer/httpRequestHandler.h"
-#include "httpServer/httpRequestRouter.h"
+#include <httpServer/httpServer.h>
+#include <httpServer/httpRequestHandler.h>
+#include <httpServer/httpRequestRouter.h>
 #include "medialibraryhandler.h"
 #include "../struct/ConnectionChangedSignal.h"
 #include "../tool/videoformat.h"
